src/nonlinearshrinkage.jl

```diff
@@ -51,39 +51,39 @@ http://www.econ.uzh.ch/static/wp/econwp264.pdf
 """
 function analytical_nonlinear_shrinkage(S::AbstractMatrix{<:Real},
                                         n::Int, p::Int, est_mean::Bool;
-                                        decomp::Union{Nothing,Eigen}=nothing)
+                                        decomp::Eigen=eigen(S))
```

Comment:
apparently the tests rely on being able to pas `nothing` so reverting this
```suggestion
                                        decomp::Union{Nothing,Eigen}=nothing)
```